screening/report_datastore_content.go

```diff
@@ -16,12 +19,36 @@ func (protocol *Protocol) handleReportDataStoreContent(packet nex.PacketInterfac
 		return
 	}
 
-	globals.Logger.Warning("Screening::ReportDataStoreContent STUBBED")
-
 	request := packet.RMCMessage()
 	callID := request.CallID
+	parameters := request.Parameters
+	endpoint := packet.Sender().Endpoint()
+	parametersStream := nex.NewByteStreamIn(parameters, endpoint.LibraryVersions(), endpoint.ByteStreamSettings())
+
+	var pContentParam screening_types.ScreeningDataStoreContentParam
+	var pViolationParam screening_types.ScreeningUGCViolationParam
+
+	err := pContentParam.ExtractFrom(parametersStream)
+	if err != nil {
+		_, rmcError := protocol.ReportDataStoreContent(fmt.Errorf("failed to read pContentParam from parameters. %s", err.Error()), packet, callID, pContentParam, pViolationParam)
```

Comment:
```suggestion
		_, rmcError := protocol.ReportDataStoreContent(fmt.Errorf("Failed to read pContentParam from parameters. %s", err.Error()), packet, callID, pContentParam, pViolationParam)
```

These extraction errors (here in the method and in the types) should all be capitalized, considering there will be periods before the beginning of these phrases

Reply:
That's actually something I keep forgetting to bring up. Technically, the way we've been doing it results in `error strings should not be capitalized (ST1005) go-staticcheck` from the linter. Currently we have over 2,000 linter warnings from this alone:

```
golangci-lint run | grep ST1005 | wc -l
    2503
```

Is this a convention we want to change to align with the linter, or just disable this linter check?

Reply:
I'm not against it but we should rewrite it everywhere if we're going with this, and use proper semantics (like replacing periods with `:`):

```
failed to read pContentParam from parameters: %s
```

Reply:
thats fair. ill correct it to what we have everywhere else for now and we can update it in another PR, seems out of scope for this one